.golangci.yml

```diff
@@ -5,6 +5,12 @@ run:
     - pkg/k8s/
     - tools/generator
 
+# all available settings of specific linters
+linters-settings:
+  misspell:
+    ignore-words:
+      - noteable
```

Comment:
notable?

Reply:
There is a miss spelling error of `MergeCommentEvent` struct in [pkg/server/biz/scm/gitlab/gitlab.go#L311-L350](https://github.com/caicloud/cyclone/blob/master/pkg/server/biz/scm/gitlab/gitlab.go#L311-L350), this struct is copied from Gitlab. It can not be fixed, so we have to ignore this error.

Reply:
This field maybe is useless for us, could we delete it?

Reply:
Have deleted